### ISSUE TITLE
Redundant padding in css causing content shift

### ISSUE THREAD

**Issue by lightsing** [apachecn/sklearn-doc-zh#257]
`padding-right` in [nature.css](http://sklearn.apachecn.org/cn/0.19.0/_static/nature.css) (line 255) causing `div.content-wrapper` shift left.

![wrong padding image](https://pbs.twimg.com/media/DQq-CJwVAAEELbK.jpg:large)

**Comment by wangyangting** [comment]
谢谢反馈！~~~ 照着你的方法，试着把首页这个多余的空白该删掉后，下部分内容居中，上部分内容没动，好尴尬，整体效果就没对齐了！~~~ ps: 不怎么会操作 CSS - - 、
可以的话，麻烦说下详细步骤，谢谢你 。。。

**Comment by wizardforcel** [comment]
@wangyangting 官方就是上下都居中的效果，不知道你改动哪儿了。

![](https://user-images.githubusercontent.com/5080126/34402978-2943a42a-ebe0-11e7-8c9c-e96b5ee25d4a.jpg)

**Comment by chenyyx** [comment]
好像是因为在里边我们添加那个校验者和翻译者的条条导致的，哈哈

**Comment by DataMonk2017** [comment]
nature.css 255
div.content-wrapper {
padding-right: 210px; #这一行删掉应该就没问题了
}
chrome, firefox 测试了有效
